Formula/w/wthrr.rb

```diff
@@ -0,0 +1,21 @@
+class Wthrr < Formula
+  desc "Weather companion for the terminal. Rust app"
```

Comment:
```suggestion
  desc "Weather companion for the terminal"
```
Being written in Rust is an implementation detail; let's leave that out of the description, thanks.